### PR TITLE
Add workflow dispatch to prod

### PR DESCRIPTION
**Pull request by AshakaE**
# workflow dispatch

_Manual deploy code to Prod_

## How should this be tested?

1. Github actions panel

closes https://github.com/EveripediaNetwork/issues/issues/443

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    steps:
+      - name: Deploy to production
+        uses: appleboy/ssh-action@master
+        with: 
+          host: ${{ secrets.HOST_PROD }}
+          username: ${{ secrets.USERNAME_PROD }}
+          key: ${{ secrets.KEY_PROD }}
+          port: ${{ secrets.PORT_PROD }}
+          script: |
+            cd vintage
```
Comment (kesar) [inline .github/workflows/deploy-prod.yml:17]:
```suggestion
            cd ep-api
```

Comment (AshakaE) [reply .github/workflows/deploy-prod.yml]: Testing 🤦

File: .github/workflows/deploy-prod.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          host: ${{ secrets.HOST_PROD }}
+          username: ${{ secrets.USERNAME_PROD }}
+          key: ${{ secrets.KEY_PROD }}
+          port: ${{ secrets.PORT_PROD }}
+          script: |
+            cd ep-api
+            git pull
+            yarn
+            yarn build
+            pm2 update
```
Comment (Royal-lobster) [inline .github/workflows/deploy-prod.yml:21]:
```suggestion
            pm2 update
            
```

File: .gitignore [hunk 3]
```diff
@@ -398,3 +398,4 @@ Temporary Items
 .env.dev
 dist
 .vscode
+ngrok.yml
```
Comment (Royal-lobster) [inline .gitignore:401]:
```suggestion
ngrok.yml

```

**Pull request merged**